docs/profiling.md

```diff
@@ -12,12 +12,22 @@ GPU and TPU. The end result looks something like this:
 
 ### Installation
 
-Install specific nightly versions of TensorBoard, TensorBoard profiler, TensorFlow, as well as the What-If Tool TensorBoard, as follows:
+The TensorBoard profiler is only available with the version of TensorBoard
+bundled with TensorFlow.
+
 ```shell
-# Profiling is known to work with the following packages.
-pip install --upgrade tb-nightly==2.5.0a20201203 tbp-nightly==2.4.0a20201203 tf-nightly==2.5.0.dev20201203 tensorboard-plugin-wit==1.7.0
+pip install tensorflow tbp-nightly
```

Comment:
Perhaps it's still worth identifying a known-good tbp-nightly release?

Reply:
Good point. I noted the tbp-nightly and tensorflow versions in the parenthetical statement below. I'm also trying to put together end-to-end Cloud TPU VM testing for this, although I don't think I can test that the trace viewer loads correct, just that TB can start. I also wanna change these instructions to use tensorflow-plugin-profile once 2.5.0 is released.